docs/stories/getting-started/usage-examples/top-app-bar-as-container.stories.js

```diff
@@ -0,0 +1,104 @@
+import '@vonage/vwc-side-drawer';
+import '@vonage/vwc-top-app-bar';
+import '@vonage/vwc-top-app-bar-fixed';
+import '@vonage/vwc-icon';
+import '@vonage/vwc-layout';
+import '@vonage/vwc-list';
+import '@vonage/vwc-list/vwc-list-item';
+import '@vonage/vwc-list/vwc-list-expansion-panel';
+import { html } from 'lit-element';
+import { spread } from '@open-wc/lit-helpers';
+
+export default {
+	title: 'Getting Started/Usage Examples/Top App Bar',
+};
+
+const style = html`
+	<style>
+		.sb-show-main.sb-main-padded {
+			padding: 0;
+		}
+		p {
+			color: "#C0C0C0";
+			font-weight: "bold";
+		}
+		vwc-side-drawer#side-drawer{
+			height: calc(100vh - 64px);
```

Comment:
because this is an example that we want ppl to understand what we did I think it will we nice to add in comment something like this:
/*side drawer initially is `100vh` but combined with top bar - the height of the top bar need to be reduced*/